diff --git a/makefile b/makefile
index a319615..4d56ade 100644
--- a/makefile
+++ b/makefile
@@ -8,8 +8,11 @@ GOOS?=$(shell go env GOOS)
 GO_SPACE?=$(CURDIR)
 GOTEMPPATH?=$(GO_SPACE)/build/private
 GOTEMPCOPYPATH?=$(GOTEMPPATH)/src/github.com/aws/session-manager-plugin
-GOPATH:=$(GOTEMPPATH):$(GO_SPACE)/vendor:$(GOPATH)
-
+ifeq (,$(findstring Windows,$(OS)))
+	GOPATH:=$(GOTEMPPATH):$(GO_SPACE)/vendor:$(GOPATH)
+else
+	GOPATH:=$(GOTEMPPATH);$(GO_SPACE)/vendor;$(GOPATH)
+endif
 export GOPATH
 export GO_SPACE
 export GO111MODULE=auto
@@ -119,7 +122,7 @@ build-darwin-amd64: checkstyle copy-src pre-build
     		$(GO_SPACE)/src/ssmcli-main/main.go
 
 .PHONY: build-windows-amd64
-build-windows-amd64: checkstyle copy-src pre-build
+build-windows-amd64: copy-src pre-build
 	@echo "Build for windows platform"
 	GOOS=windows GOARCH=amd64 $(GO_BUILD) -ldflags "-s -w" -o $(GO_SPACE)/bin/windows_amd64_plugin/session-manager-plugin.exe -v \
 		$(GO_SPACE)/src/sessionmanagerplugin-main/main.go
@@ -127,7 +130,7 @@ build-windows-amd64: checkstyle copy-src pre-build
     		$(GO_SPACE)/src/ssmcli-main/main.go
 
 .PHONY: build-windows-386
-build-windows-386: checkstyle copy-src pre-build
+build-windows-386: copy-src pre-build
 	@echo "Build for windows platform"
 	GOOS=windows GOARCH=386 $(GO_BUILD) -ldflags "-s -w" -o $(GO_SPACE)/bin/windows_386_plugin/session-manager-plugin.exe -v \
 		$(GO_SPACE)/src/sessionmanagerplugin-main/main.go
